### DISCUSSION TITLE
Add "href" props to Qpagination

### DISCUSSION THREAD

**Discussion by webgoto** [quasarframework/quasar#15947] [Ideas / Proposals]
I want to make pagination link with "a" tag.

`<a href="https://aaaa.com/?page=4">4</a>`

Is there a way to create a link "a" tag?
I never use vue-router.
I can't make it with the "to" props.

**Comment by metalsadman** [comment]
just update your api call to that url when a pagination link is clicked

**Reply by webgoto** [reply]
Then it's not an 'a' tag.

**Comment by pdanpdan** [comment]
`to-fn` prop

**Reply by webgoto** [reply]
Thanks for the message.
I believe that this prop won't work without vue-router.

**Reply by pdanpdan** [reply]
Yes, of course - all to-like props only work with vue-router

**Reply by webgoto** [reply]
I don't want to use vue-router.
So please allow developers to create 'a' tags by adding "href" props.
Or want to apply arbitrary props to qBtn.
